spdx-utils/src/main/kotlin/Extensions.kt

```diff
@@ -32,7 +32,7 @@ import org.ossreviewtoolkit.spdx.SpdxExpression.Strictness
 /**
  * Return a string of hexadecimal digits representing the bytes in the array.
  */
-fun ByteArray.toHexString(): String = joinToString("") { String.format("%02x", it) }
+fun ByteArray.toHexString(): String = joinToString("") { String.format(null, "%02x", it) }
```

Comment:
The `Locale` class has a special constant `ROOT`, which should be used instead of *null*.

Reply:
Thanks, I wasn't aware of that constant.